EnglishFilter/sections/general_extensions.txt

```diff
@@ -236,6 +236,8 @@ jav688.com#%#//scriptlet("set-constant", "D4zz", "noopFunc")
 movierulzfree.is#%#//scriptlet('prevent-window-open')
 ! https://github.com/AdguardTeam/AdguardFilters/issues/72810
 5movierulzfree.me#%#//scriptlet('prevent-window-open')
+! https://github.com/AdguardTeam/AdguardFilters/issues/72616
+voe.*#%#//scriptlet("prevent-window-open")
```

Comment:
why `*`?

Reply:
There are more domains I am assuming.

Reply:
It seems that it breaks download button.
Download button is also broken by the rule which I removed already - https://github.com/AdguardTeam/AdguardFilters/commit/b6a8913b8ffbc8788094d40223b70a51d90fa5ff

This rule seems to works fine:
```
voe.sx#%#//scriptlet('abort-current-inline-script', 'EventTarget.prototype.addEventListener', '/location[\s\S]*?getElementsByTagName/')
```
at least until they will change something again.

Reply:
They change their code every day

Reply:
You can enable AdGuard Popup Blocker, it blocks these popups.

Reply:
`voe.*#%#//scriptlet("prevent-window-open")` This one does all the job

Reply:
`voe.*#%#//scriptlet("prevent-window-open") This one does all the job`
It breaks download button.

Reply:
hmm..sensitive too

Reply:
>AdGuard Pop-up Blocker is quite useless. In most of the sites, it fails only some sites it blocks. No offense.

Okay, but it works fine in this case, so I don't see any reason to not use it.

Reply:
This site it does the job so yeah